eng/targets/Settings.props

```diff
@@ -20,6 +20,7 @@
     <CommonExtensionInstallationRoot>CommonExtensions</CommonExtensionInstallationRoot>
     <LanguageServicesExtensionInstallationFolder>Microsoft\VBCSharp\LanguageServices</LanguageServicesExtensionInstallationFolder>
 
+    <!-- RestoreUseStaticGraphEvaluation will cause prebuilts when building source-only. -->
     <RestoreUseStaticGraphEvaluation>true</RestoreUseStaticGraphEvaluation>
```

Comment:
Do I understand correctly that it's still useful to have this here because the Build.props wouldn't be used during `dotnet build` for example, that's only used during `build.cmd` and similar, right? Would it make sense to import Build.props from here to avoid duplication?

Reply:
Correct. The setting in Build.props controls how Arcade restores the solution vs this setting controls how non Arcade invocations like `dotnet X` restore. Build.props shouldn't get imported anywhere else than by Arcade for different reasons: separation of concerns but also some potential settings in Build.props could cause incorrect behavior in the non Arcade build entry point.